docs/python_api/datasource/gwas_catalog/_gwas_catalog.md

```diff
@@ -6,3 +6,17 @@ title: GWAS Catalog
   <img width="100" height="100" src="../../../../assets/imgs/GWAS_Catalog_circle_178x178.png">
   <h1>GWAS Catalog</h1>
 </div>
+
+The [GWAS Catalog](https://www.ebi.ac.uk/gwas/) is a comprehensive resource that aims to provide a curated collection of Genome-Wide Association Studies (GWAS) across various traits and diseases in humans.
+
+It serves as a valuable repository of genetic associations identified in diverse populations, offering insights into the genetic basis of complex traits and diseases.
+
+We rely on the GWAS Catalog for a rich source of genetic associations, utilizing the data for analysis and interpretation.
+
+For detailed information on specific genetic associations, their significance, and associated studies, refer to the [GWAS Catalog](https://www.ebi.ac.uk/gwas/).
+
+Within our analyses, we leverage two different types of studies from the GWAS Catalog:
+
+1. **Studies with (full) GWAS summary stats:** currently 25,569 studies.
+
+2. **Studies with top hits only - GWAS curated studies:** currently ~50,000 studies. Called "Associations".
```

Comment:
Same as in Finngen.

Reply:
What do you mean?

Reply:
Same comment I had for the study, and trait counts in the [finngen docs](https://github.com/opentargets/genetics_etl_python/pull/362/files#diff-c6b1776eb7e22b7055441e8c1b4240b6cb7e2de33b9bd26ef1fa1349e2da66ed) also